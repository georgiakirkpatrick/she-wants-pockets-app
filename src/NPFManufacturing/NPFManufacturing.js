import React from 'react'
import FormPage from '../FormPage/FormPage'
import FormButton from '../FormButton/FormButton'
import FormCheckboxSection from '../FormCheckboxSection/FormCheckboxSection'
import FormDropdown from '../FormDropdown/FormDropdown'
import FormPopUp from '../FormPopUp/FormPopUp'
import FormPromptWithSub from '../FormPromptWithSub/FormPromptWithSub'
import NPFFooter from '../NPFFooter/NPFFooter'
import FormTextInput from '../FormTextInput/FormTextInput'
import FormUrlInput from '../FormUrlInput/FormUrlInput'
import formData from '../FORM_DATA'
import './NPFManufacturing.css'
import FormFieldset from '../FormFieldset/FormFieldset';

const NPFManufacturing = props => {
    const sewingChangeInput = event => {
        const sewingFactory = {...props.manuState.sewFact}
        sewingFactory[event.target.name] = event.target.value
        props.manuState.setSewFact(sewingFactory)
    }

    const sewingPopUpStatus = () => {
        if (props.manuState.sewFactPopUp === true) {
            return 'FormPopUp__pop-up active'
        }
        return 'FormPopUp__pop-up'
    }

    const handleSewingClose = () => { 
        props.manuState.setSewFactPopUp(false)
    }

    const newSewingChangeInput = event => {
        const newSewFactFields = {...props.manuState.newSewFact}
        newSewFactFields[event.target.name] = event.target.value
        props.manuState.setNewSewFact(newSewFactFields)
    }

    // CUTTING FACTORY
    const cuttingChangeInput = event => {
        const cuttingFactory = {...props.manuState.cutFact}
        cuttingFactory[event.target.name] = event.target.value
        props.manuState.setCutFact(cuttingFactory)
    }

    const cuttingPopUpStatus = () => {
        if (props.manuState.cutFactPopUp === true) {
            return 'FormPopUp__pop-up active'
        }
        return 'FormPopUp__pop-up'
    }

    const handleCuttingClose = () => {
        props.manuState.setCutFactPopUp(false)
    }

    const newCuttingChangeInput = event => {
        const newCutFactFields = {...props.manuState.newCutFact}
        newCutFactFields[event.target.name] = event.target.value
        props.manuState.setNewCutFact(newCutFactFields)
    }

    const certPopUpStatus = () => {
        if (props.manuState.certPopUp === true) {
            return 'FormPopUp__pop-up active'
        }
        return 'FormPopUp__pop-up'
    }

    const handleCertClose = () => {
        props.manuState.setCertPopUp(false)
    }

    const certificationChange = event => {
        props.manuState.setCertChecks({...props.manuState.certChecks, [event.target.id]: !props.manuState.certChecks[event.target.id]})
    }

    const newCertChangeInput = event => {
        const newCertFields = {...props.manuState.newCert}
        newCertFields[event.target.name] = event.target.value
        props.manuState.setNewCert(newCertFields)
    }

    // ADDITIONAL NOTES
    const additionalNotesChange = event => {
        const newNotes = event.target.value
        props.manuState.setAdditionalNotes(newNotes)
    }

    return (
        <div id={formData.manufacturing.id}>
            <FormPage title={formData.manufacturing.pageTitle}>
                <FormPromptWithSub 
                    prompt={formData.manufacturing.pagePrompts[0].prompt}
                    promptSubtitle={formData.manufacturing.pagePrompts[0].promptSubtitle}
                />
                <FormFieldset 
                    prompt={formData.manufacturing.pagePrompts[1].prompt}
                    subprompt={formData.manufacturing.pagePrompts[1].subprompt}
                >
                    <FormDropdown
                        id={formData.manufacturing.location.name}
                        name={formData.manufacturing.location.name}
                        prompt={formData.manufacturing.location.prompt}
                        value={props.manuState.sewFact.location}
                        options={props.countries}
                        change={event => sewingChangeInput(event)} 
                    />
                    <FormDropdown
                        id={formData.manufacturing.factory.name}
                        name={formData.manufacturing.factory.name}
                        prompt={formData.manufacturing.factory.prompt}
                        value={props.manuState.sewFact.factory}
                        options={props.factories}
                        change={event => sewingChangeInput(event)} 
                    />
                    <FormButton buttonText={formData.manufacturing.buttonText[0]} click={() => props.manuState.setSewFactPopUp(true)} />
                </FormFieldset>

                <FormPopUp
                    id={formData.manufacturing.addSewing.id} 
                    status={sewingPopUpStatus()}
                    title={formData.manufacturing.addSewing.pageTitle}
                    click={() => handleSewingClose()}
                    buttonText={formData.manufacturing.addSewing.submitButton}
                >
                    <FormPromptWithSub prompt={formData.manufacturing.addSewing.prompt} promptSubtitle='' />
                    <FormTextInput 
                        id={formData.manufacturing.addSewing.factoryName.id}
                        name={formData.manufacturing.addSewing.factoryName.name}
                        prompt={formData.manufacturing.addSewing.factoryName.prompt} 
                        value={props.manuState.newSewFact.sewingFactoryName}
                        change={event => newSewingChangeInput(event)} 
                    />
                    <FormDropdown
                        id={formData.manufacturing.addSewing.factoryLocation.id}
                        name={formData.manufacturing.addSewing.factoryLocation.id}
                        prompt={formData.manufacturing.addSewing.factoryLocation.prompt}
                        value={props.manuState.newSewFact.sewingFactoryLocation}
                        options={props.countries}
                        change={event => newSewingChangeInput(event)} 
                    />
                    <FormUrlInput 
                        id={formData.manufacturing.addSewing.factoryWebsite.id} 
                        name={formData.manufacturing.addSewing.factoryWebsite.id} 
                        prompt={formData.manufacturing.addSewing.factoryWebsite.prompt}
                        value={props.manuState.newSewFact.sewingFactoryWebsite}
                        change={event => newSewingChangeInput(event)}
                    />
                    <FormTextInput
                        id={formData.manufacturing.addSewing.factoryNotes.id}
                        prompt={formData.manufacturing.addSewing.factoryNotes.prompt}
                        value={props.manuState.newSewFact.sewingFactoryNotes}
                        change={event => newSewingChangeInput(event)}
                    />
                </FormPopUp>
                
                <FormFieldset 
                    prompt={formData.manufacturing.pagePrompts[2].prompt}
                    subprompt={formData.manufacturing.pagePrompts[2].subprompt}
                >
                    <FormDropdown
                        id={formData.manufacturing.location.name}
                        name={formData.manufacturing.location.name}
                        prompt={formData.manufacturing.location.prompt}
                        value={props.manuState.cutFact.location}
                        options={props.countries}
                        change={event => cuttingChangeInput(event)} 
                    />
                    <FormDropdown
                        id={formData.manufacturing.factory.name}
                        name={formData.manufacturing.factory.name}
                        prompt={formData.manufacturing.factory.prompt}
                        value={props.manuState.cutFact.factory}
                        options={props.factories}
                        change={event => cuttingChangeInput(event)} 
                    />
                    <FormButton buttonText={formData.manufacturing.buttonText[0]} click={() => props.manuState.setCutFactPopUp(true)}/>
                </FormFieldset>

                <FormPopUp
                    id={formData.manufacturing.addCutting.id} 
                    status={cuttingPopUpStatus()}
                    title={formData.manufacturing.addCutting.pageTitle}
                    click={() => handleCuttingClose()}
                    buttonText={formData.manufacturing.addCutting.submitButton}
                >
                    <FormPromptWithSub prompt={formData.manufacturing.addCutting.prompt} promptSubtitle='' />
                    <FormTextInput 
                        id={formData.manufacturing.addCutting.factoryName.id}
                        name={formData.manufacturing.addCutting.factoryName.id}
                        prompt={formData.manufacturing.addCutting.factoryName.prompt} 
                        value={props.manuState.newCutFact.cuttingFactoryName}
                        change={event => newCuttingChangeInput(event)}
                    />
                    <FormDropdown
                        id={formData.manufacturing.addCutting.factoryLocation.id}
                        name={formData.manufacturing.addCutting.factoryLocation.id}
                        prompt={formData.manufacturing.addCutting.factoryLocation.prompt}
                        value={props.manuState.newCutFact.cuttingFactoryLocation}
                        options={props.countries}
                        change={event => newCuttingChangeInput(event)}
                    />
                    <FormUrlInput
                        id={formData.manufacturing.addCutting.factoryWebsite.id}
                        name={formData.manufacturing.addCutting.factoryWebsite.id}
                        prompt={formData.manufacturing.addCutting.factoryWebsite.prompt}
                        value={props.manuState.newCutFact.cuttingFactoryWebsite}
                        change={event => newCuttingChangeInput(event)}
                    />
                    <FormTextInput
                        id={formData.manufacturing.addCutting.factoryNotes.id}
                        name={formData.manufacturing.addCutting.factoryNotes.id}
                        prompt={formData.manufacturing.addCutting.factoryNotes.prompt}
                        value={props.manuState.newCutFact.cuttingFactoryNotes}
                        change={event => newCuttingChangeInput(event)}
                    />
                </FormPopUp>

                <FormFieldset
                    prompt={formData.manufacturing.pagePrompts[3].prompt}
                    subprompt={formData.manufacturing.pagePrompts[3].subprompt}
                >
                    <FormCheckboxSection  
                        options={formData.manufacturing.certifications.options} 
                        check={props.manuState.certChecks}
                        change={event => certificationChange(event)}
                    />
                    <FormButton buttonText={formData.manufacturing.buttonText[1]} click={() => props.manuState.setCertPopUp(true)} />
                </FormFieldset>

                <FormPopUp
                    id={formData.manufacturing.certifications.id} 
                    status={certPopUpStatus()}
                    title={formData.manufacturing.newCertification.title}
                    click={() => handleCertClose()}
                    buttonText={formData.manufacturing.newCertification.submitButton}
                >
                    <FormPromptWithSub prompt='Add a certification' promptSubtitle='' />
                    <FormTextInput 
                        id='newCertName'
                        name='newCertName'
                        prompt={formData.manufacturing.newCertification.certName.prompt} 
                        value={props.manuState.newCert.newCertName}
                        change={event => newCertChangeInput(event)} 
                    />
                    <FormUrlInput
                        id='newCertWebsite'
                        name='newCertWebsite'
                        prompt={formData.manufacturing.newCertification.certWebsite.prompt}
                        value={props.manuState.newCert.newCertWebsite}
                        change={event => newCertChangeInput(event)}
                    />
                </FormPopUp>

                <FormFieldset
                    prompt={formData.manufacturing.additionalNotes.prompt}
                    subprompt={formData.manufacturing.additionalNotes.subprompt}
                >
                    <FormTextInput
                        id={formData.manufacturing.additionalNotes.id}
                        name={formData.manufacturing.additionalNotes.id}
                        value={props.manuState.additionalNotes}
                        change={event => additionalNotesChange(event)}
                    />
                </FormFieldset>
            </FormPage>
            <NPFFooter buttons='prevNext' previousButton={() => props.setPage(props.currentPage - 1)} nextButton={() => props.setPage(props.currentPage + 1)} />
        </div>
    )  
}

export default NPFManufacturing;