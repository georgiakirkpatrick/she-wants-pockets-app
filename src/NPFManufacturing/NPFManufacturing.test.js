import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import NPFManufacturing from './NPFManufacturing'

describe('NPFManufacturing', () => {
    it ('renders without crashing', () => {
        const div = document.createElement('div')
        ReactDOM.render(<NPFManufacturing />, div)
        ReactDOM.unmountComponentAtNode(div)
    })

    it ('render the UI as expected', () => {
        const tree = renderer
            .create(<NPFManufacturing />)
            .toJSON()
        expect(tree).toMatchSnapshot()
    })
})