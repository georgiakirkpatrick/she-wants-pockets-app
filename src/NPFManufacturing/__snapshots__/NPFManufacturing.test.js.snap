// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NPFManufacturing render the UI as expected 1`] = `
<div
  id="manufacturing"
>
  <section
    className="FormPage"
  >
    <h1>
      Manufacturing
    </h1>
    <form>
      <div
        className="FormPromptWithSub"
      >
        <p
          className="FormPromptWithSub__prompt"
        >
          Enter the manufacturing information provided on the product page.
        </p>
        <p
          className="FormPromptWithSub__subtitle"
        >
          
        </p>
      </div>
      <fieldset
        className="FormFieldset"
      >
        <p>
          Sewing and finishing
        </p>
        <div
          className="FormDropdown"
        >
          <label
            htmlFor="location"
          >
            Location
          </label>
          <select
            name="location"
            onChange={[Function]}
            value=""
          />
        </div>
        <div
          className="FormDropdown"
        >
          <label
            htmlFor="factory"
          >
            Factory
          </label>
          <select
            name="factory"
            onChange={[Function]}
            value=""
          />
        </div>
        <button
          className="FormButton"
          onClick={[Function]}
          type="button"
        >
          ADD A FACTORY
        </button>
      </fieldset>
      <section
        className="FormPopUp__pop-up"
        id="addSewing"
      >
        <button
          className="FormPopUp__close-button"
          onClick={[Function]}
          type="button"
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-times fa-w-11 fa-lg "
            data-icon="times"
            data-prefix="fas"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 352 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M242.72 256l100.07-100.07c12.28-12.28 12.28-32.19 0-44.48l-22.24-22.24c-12.28-12.28-32.19-12.28-44.48 0L176 189.28 75.93 89.21c-12.28-12.28-32.19-12.28-44.48 0L9.21 111.45c-12.28 12.28-12.28 32.19 0 44.48L109.28 256 9.21 356.07c-12.28 12.28-12.28 32.19 0 44.48l22.24 22.24c12.28 12.28 32.2 12.28 44.48 0L176 322.72l100.07 100.07c12.28 12.28 32.2 12.28 44.48 0l22.24-22.24c12.28-12.28 12.28-32.19 0-44.48L242.72 256z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </button>
        <h1>
          New Factory
        </h1>
        <div
          className="FormPromptWithSub"
        >
          <p
            className="FormPromptWithSub__prompt"
          >
            Add a sewing and finishing factory
          </p>
          <p
            className="FormPromptWithSub__subtitle"
          >
            
          </p>
        </div>
        <div
          className="FormText"
        >
          <label
            htmlFor="sewingFactoryName"
          >
            Factory name
          </label>
          <input
            defaultValue=""
            id="sewingFactoryName"
            name=""
            onChange={[Function]}
            type="text"
          />
        </div>
        <div
          className="FormDropdown"
        >
          <label
            htmlFor="sewingFactoryLocation"
          >
            Location
          </label>
          <select
            name="sewingFactoryLocation"
            onChange={[Function]}
            value=""
          />
        </div>
        <div
          className="FormUrlInput"
        >
          <label
            htmlFor="sewingFactoryWebsite"
          >
            Website
          </label>
          <input
            defaultValue=""
            id="sewingFactoryWebsite"
            name="sewingFactoryWebsite"
            onChange={[Function]}
            type="url"
          />
        </div>
        <div
          className="FormText"
        >
          <label
            htmlFor="sewingFactoryNotes"
          >
            Notes
          </label>
          <input
            defaultValue=""
            id="sewingFactoryNotes"
            name=""
            onChange={[Function]}
            type="text"
          />
        </div>
        <button
          className="FormButton"
          onClick={[Function]}
          type="button"
        >
          SUBMIT FACTORY
        </button>
      </section>
      <fieldset
        className="FormFieldset"
      >
        <p>
          Fabric cutting
        </p>
        <div
          className="FormDropdown"
        >
          <label
            htmlFor="location"
          >
            Location
          </label>
          <select
            name="location"
            onChange={[Function]}
            value=""
          />
        </div>
        <div
          className="FormDropdown"
        >
          <label
            htmlFor="factory"
          >
            Factory
          </label>
          <select
            name="factory"
            onChange={[Function]}
            value=""
          />
        </div>
        <button
          className="FormButton"
          onClick={[Function]}
          type="button"
        >
          ADD A FACTORY
        </button>
      </fieldset>
      <section
        className="FormPopUp__pop-up"
        id="addCutting"
      >
        <button
          className="FormPopUp__close-button"
          onClick={[Function]}
          type="button"
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-times fa-w-11 fa-lg "
            data-icon="times"
            data-prefix="fas"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 352 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M242.72 256l100.07-100.07c12.28-12.28 12.28-32.19 0-44.48l-22.24-22.24c-12.28-12.28-32.19-12.28-44.48 0L176 189.28 75.93 89.21c-12.28-12.28-32.19-12.28-44.48 0L9.21 111.45c-12.28 12.28-12.28 32.19 0 44.48L109.28 256 9.21 356.07c-12.28 12.28-12.28 32.19 0 44.48l22.24 22.24c12.28 12.28 32.2 12.28 44.48 0L176 322.72l100.07 100.07c12.28 12.28 32.2 12.28 44.48 0l22.24-22.24c12.28-12.28 12.28-32.19 0-44.48L242.72 256z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </button>
        <h1>
          New Factory
        </h1>
        <div
          className="FormPromptWithSub"
        >
          <p
            className="FormPromptWithSub__prompt"
          >
            Add a cutting factory
          </p>
          <p
            className="FormPromptWithSub__subtitle"
          >
            
          </p>
        </div>
        <div
          className="FormText"
        >
          <label
            htmlFor="cuttingFactoryName"
          >
            Factory name
          </label>
          <input
            defaultValue=""
            id="cuttingFactoryName"
            name="cuttingFactoryName"
            onChange={[Function]}
            type="text"
          />
        </div>
        <div
          className="FormDropdown"
        >
          <label
            htmlFor="cuttingFactoryLocation"
          >
            Location
          </label>
          <select
            name="cuttingFactoryLocation"
            onChange={[Function]}
            value=""
          />
        </div>
        <div
          className="FormUrlInput"
        >
          <label
            htmlFor="cuttingFactoryWebsite"
          >
            Website
          </label>
          <input
            defaultValue=""
            id="cuttingFactoryWebsite"
            name="cuttingFactoryWebsite"
            onChange={[Function]}
            type="url"
          />
        </div>
        <div
          className="FormText"
        >
          <label
            htmlFor="cuttingFactoryNotes"
          >
            Notes
          </label>
          <input
            defaultValue=""
            id="cuttingFactoryNotes"
            name="cuttingFactoryNotes"
            onChange={[Function]}
            type="text"
          />
        </div>
        <button
          className="FormButton"
          onClick={[Function]}
          type="button"
        >
          SUBMIT FACTORY
        </button>
      </section>
      <fieldset
        className="FormFieldset"
      >
        <p>
          Does the fabric have any of the following certifications?
        </p>
        <div
          className="FormCheckboxSection"
        >
          <div
            className="empty"
          />
          <div
            className="FormCheckboxSection__options-container"
          >
            <div
              className="FormCheckboxSection__option"
            >
              <input
                id={1}
                name={1}
                onChange={[Function]}
                type="checkbox"
              />
              <label
                htmlFor={1}
              >
                BlueSign
              </label>
            </div>
            <div
              className="FormCheckboxSection__option"
            >
              <input
                id={2}
                name={2}
                onChange={[Function]}
                type="checkbox"
              />
              <label
                htmlFor={2}
              >
                OEKO-TEX
              </label>
            </div>
          </div>
        </div>
        <button
          className="FormButton"
          onClick={[Function]}
          type="button"
        >
          ADD A CERTIFICATION
        </button>
      </fieldset>
      <section
        className="FormPopUp__pop-up"
        id={1}
      >
        <button
          className="FormPopUp__close-button"
          onClick={[Function]}
          type="button"
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-times fa-w-11 fa-lg "
            data-icon="times"
            data-prefix="fas"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 352 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M242.72 256l100.07-100.07c12.28-12.28 12.28-32.19 0-44.48l-22.24-22.24c-12.28-12.28-32.19-12.28-44.48 0L176 189.28 75.93 89.21c-12.28-12.28-32.19-12.28-44.48 0L9.21 111.45c-12.28 12.28-12.28 32.19 0 44.48L109.28 256 9.21 356.07c-12.28 12.28-12.28 32.19 0 44.48l22.24 22.24c12.28 12.28 32.2 12.28 44.48 0L176 322.72l100.07 100.07c12.28 12.28 32.2 12.28 44.48 0l22.24-22.24c12.28-12.28 12.28-32.19 0-44.48L242.72 256z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </button>
        <h1>
          New Certification
        </h1>
        <div
          className="FormPromptWithSub"
        >
          <p
            className="FormPromptWithSub__prompt"
          >
            Add a certification
          </p>
          <p
            className="FormPromptWithSub__subtitle"
          >
            
          </p>
        </div>
        <div
          className="FormText"
        >
          <label
            htmlFor="newCertName"
          >
            Certification name
          </label>
          <input
            defaultValue=""
            id="newCertName"
            name="newCertName"
            onChange={[Function]}
            type="text"
          />
        </div>
        <div
          className="FormUrlInput"
        >
          <label
            htmlFor="newCertWebsite"
          >
            Website
          </label>
          <input
            defaultValue=""
            id="newCertWebsite"
            name="newCertWebsite"
            onChange={[Function]}
            type="url"
          />
        </div>
        <button
          className="FormButton"
          onClick={[Function]}
          type="button"
        >
          SUBMIT CERTIFICATION
        </button>
      </section>
      <fieldset
        className="FormFieldset"
      >
        <p>
          Are there any additional notes about how the product was cut, sewn, or finished?
        </p>
        <div
          className="FormText"
        >
          <label
            htmlFor="additionalNotes"
          >
            
          </label>
          <input
            defaultValue=""
            id="additionalNotes"
            name="additionalNotes"
            onChange={[Function]}
            type="text"
          />
        </div>
      </fieldset>
    </form>
  </section>
  <footer
    className="NPFFooter"
  >
    <button
      className="NPFFooter_previous"
      onClick={[Function]}
    >
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-arrow-left fa-w-14 NPFFooter_arrow"
        data-icon="arrow-left"
        data-prefix="fas"
        focusable="false"
        role="img"
        style={Object {}}
        viewBox="0 0 448 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M257.5 445.1l-22.2 22.2c-9.4 9.4-24.6 9.4-33.9 0L7 273c-9.4-9.4-9.4-24.6 0-33.9L201.4 44.7c9.4-9.4 24.6-9.4 33.9 0l22.2 22.2c9.5 9.5 9.3 25-.4 34.3L136.6 216H424c13.3 0 24 10.7 24 24v32c0 13.3-10.7 24-24 24H136.6l120.5 114.8c9.8 9.3 10 24.8.4 34.3z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
      <span>
        PREVIOUS
      </span>
    </button>
    <button
      className="NPFFooter_next"
      onClick={[Function]}
    >
      <span>
        NEXT
      </span>
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-arrow-right fa-w-14 NPFFooter_arrow"
        data-icon="arrow-right"
        data-prefix="fas"
        focusable="false"
        role="img"
        style={Object {}}
        viewBox="0 0 448 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M190.5 66.9l22.2-22.2c9.4-9.4 24.6-9.4 33.9 0L441 239c9.4 9.4 9.4 24.6 0 33.9L246.6 467.3c-9.4 9.4-24.6 9.4-33.9 0l-22.2-22.2c-9.5-9.5-9.3-25 .4-34.3L311.4 296H24c-13.3 0-24-10.7-24-24v-32c0-13.3 10.7-24 24-24h287.4L190.9 101.2c-9.8-9.3-10-24.8-.4-34.3z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
    </button>
  </footer>
</div>
`;
